### PR TITLE
7 post comment

### PR DESCRIPTION
**Pull request by Rionagrace**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -95,37 +95,99 @@ describe("/api/articles", () => {
...
+				username: "rogersop",
+				body: "loved this",
+			};
+			return request(app)
+				.post("/api/articles/50000/comments")
+				.send(comment)
+				.expect(404)
+				.then(({ body }) => {
+					expect(body.msg).toBe("article not found");
+				});
```
Comment (nc-nick-jones) [inline __tests__/app.test.js:190]: one more test for non-existent user would be good!

File: api/models/comments.models.js [hunk 2]
```diff
@@ -18,4 +19,32 @@ function selectCommentsById(article_id) {
...
\ No newline at end of file
+function insertCommentById(article_id, comment) {
+	if (!comment.username || !comment.body) {
+		return Promise.reject({
+			status: 400,
+			msg: "missing either username or body",
+		});
+	}
+	return db
+		.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
```
Comment (nc-nick-jones) [inline api/models/comments.models.js:30]: just seen new updated PR! great work

**Review by nc-nick-jones — COMMENTED** [review]
All good.  Just need the extra test mentioned!

File: __tests__/app.test.js (hunk 1 above)
Comment (Rionagrace) [reply __tests__/app.test.js:190]: added this now and added a new pr! thanks

**Pull request merged**